binding/rdf4j/src/test/resources/strstarts-function/sparql-strstarts.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE person (
+    person_id integer NOT NULL,
+    first_name text
+);
+INSERT INTO person VALUES (1, 'Mary');
+INSERT INTO person VALUES (2, 'John');
+INSERT INTO person VALUES (3, NULL);
+INSERT INTO person VALUES (4, 'James');
+
+CREATE TABLE house (
+    house_id integer NOT NULL,
```

Comment:
Please a unique constraint (or a primary key). This is standard practice and will clean your examples from distincts.